src/components/CardContainer.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from "react"
+
+import elementStyles from "styles/isomer-cms/Elements.module.scss"
+
+export const CardContainer = ({ cardTitle, children, isError, onClose }) => {
+  const [shouldDisplay, setShouldDisplay] = useState(false)
+
+  return (
+    <div
+      className={`${elementStyles.card} ${
+        !shouldDisplay && isError ? elementStyles.error : ""
+      }`}
+    >
```

Comment:
> shouldn't the error outline (1px border red) still be displayed even if the card is closed?
Do you mean that the error outline should still be displayed even when the card is OPEN? 

Currently the error outline appears only if it's closed. When the card is open, we remove the error outline since we were thinking that they would see the erroneous value when the card is open. But ya happy to discuss if this is not ideal behavior.

Sorry if the naming of the variable is confusing! Let me know